code/modules/martial_arts/martial.dm

```diff
@@ -271,6 +271,35 @@
 	new /obj/effect/decal/cleanable/ash(get_turf(src))
 	qdel(src)
 
+/obj/item/tae_clown_do_scroll
+	name = "scroll of tae clown do"
+	desc = "A scroll filled with hilarious markings. It seems to be drawings of some sort of silly martial art."
+	icon = 'icons/obj/wizard.dmi'
+	icon_state = "clownscroll"
+
+/obj/item/tae_clown_do_scroll/attack_self(mob/living/carbon/human/user as mob)
+	if(!istype(user) || !user)
+		return
+	if(user.mind && (user.mind.changeling || user.mind.vampire)) //Prevents changelings and vampires from being able to learn it
+		if(user.mind && user.mind.changeling) //Changelings
+			to_chat(user, "<span class ='warning'>We try multiple times, but we are not unable to focus on the contents of the scroll as we continue bursting out in laughter!</span>")
+			return
+		else //Vampires
+			to_chat(user, "<span class ='warning'>Your antiquated sense of humour prevents you from being able to understand the art of Tae Clown Do!</span>")
+			return
```

Comment:
```suggestion
	//Prevents changelings and vampires from being able to learn it
	if(user.mind?.changeling) //Changelings
		to_chat(user, "<span class='warning'>We try multiple times, but we are not unable to focus on the contents of the scroll as we continue bursting out in laughter!</span>")
		return
	if(user.mind?.vampire) //Vampires
		to_chat(user, "<span class='warning'>Your antiquated sense of humour prevents you from being able to understand the art of Tae Clown Do!</span>")
		return
```
`var?.something` is the same as `var && var.something`.
Best to also just make separate checks for them instead of checking them both first then doing the checks again